source/computer/fedora-setup.rst

```diff
@@ -1,17 +1,220 @@
 Fedora 配置指南
 ===============
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-01-19
+:预计花费时间: 120 分钟
+
 .. note::
 
-   本文尚未完成。
+   本节内容适用于 **Fedora 33 Workstation**\，对其他 Fedora 版本不一定适用。
+   建议用户安装时总是选择 Fedora 最新版本。如遇到问题，欢迎反馈。
+
+----
+
+安装系统
+--------
+
+.. note::
+
+   安装 Fedora 也可以参考 https://techz.io/how-to-install-fedora/
+   给出的详细步骤与图解。
+
+下载 ISO 镜像
+^^^^^^^^^^^^^
+
+访问 `Fedora 官网 <https://getfedora.org/>`__ 并下载 Fedora Workstation 镜像，
+一般选择 x86_64 版本。
+
+**Fedora 33 Workstation x86_64** 的 ISO 文件下载链接：
+
+- `官方镜像 <https://download.fedoraproject.org/pub/fedora/linux/releases/33/Workstation/x86_64/iso/Fedora-Workstation-Live-x86_64-33-1.2.iso>`__
+- `国内中科大镜像 <http://mirrors.ustc.edu.cn/fedora/releases/33/Workstation/x86_64/iso/Fedora-Workstation-Live-x86_64-33-1.2.iso>`__
+
+制作 USB 启动盘
+^^^^^^^^^^^^^^^
+
+在 Windows 下，可以使用 `Rufus <https://rufus.ie/zh_CN.html>`__ 制作
+USB 启动盘。
+
+从 USB 启动盘进入 Live 系统
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+将制作好的 USB 启动盘插入要安装 Fedora 系统的计算机上，开机启动，
+按下 F10 或 F12 进入 BIOS，并使计算机优先从 USB 盘启动。
+注意，不同型号的电脑进入 BIOS 的方法也不同，请自行查询。
+
+若计算机无法从 USB 盘启动，则可能是由于计算机的“安全启动”设置导致的，可以尝试：
+
+1. 进入 BIOS 设置
+2. 在 BIOS 设置内关闭安全启动
+
+如果尝试多次都无法正确从 USB 启动，则可能是 USB 启动盘制作失败，请尝试重新制作启动盘。
+
+开始安装
+^^^^^^^^
 
-   可参考 https://blog.seisman.info/centos7-setup/ 一文。
-   但要注意，该文基于 CentOS 7 而非 Fedora，尽管二者有相似之处。
-   目前不再建议用户使用 CentOS 7 系统。
+进入 Live 系统之后，选择“Install to Hard Drive” 即开始安装。
+
+安装程序会首先要你选择安装过程中的语言，可以选择“中文”->“简体中文（中国）”，
+然后点击下方的“继续”按钮。当然你也可以选择“English”->“English (Unite States)”。
+接下来会选择键盘布局（汉语或“English(US)”）、时区和时间（例如“亚洲-上海”）。
+
+点击“安装目的地”，选择要将系统安装到哪一块硬盘以及如何分区。
+
+在“设备选择”中，选择自己要使用哪些硬盘安装系统。如果计算机有多个磁盘，可以
+将多个磁盘都选中，被选中的磁盘会有一个“对号”符号。需要注意，不要选中 USB 启动盘。
+
+在“存储设置”中，选择“自动”让安装程序帮你进行自动分区。也可以选择“自定义”，
+但需要你了解 Linux 的分区操作。由于 Fedora 33 采用了最新的 btrfs 文件系统，
+所以对于 Fedora 33 而言，使用默认的“自动”分区即可。
+
+点击“开始安装”即进入正式安装过程。
+
+等待安装完成，点击“完成安装”，并重启计算机。
+重启计算机时，记得拔出 USB 启动盘，以免再次进入 USB 安装镜像。
+
+重启后会进入欢迎页面，需要添加账户，用户名只能是英文。
+
+系统软件
+--------
+
+Fedora 使用 ``dnf`` 包管理器来安装、卸载和管理软件包。
 
 .. note::
```

Comment:
Shall we move this note to the end of the subsection?
It seems the babies need to run `sudo dnf makecache` when they change the source.

Reply:
> Shall we move this note to the end of the subsection?
> It seems the babies need to run `sudo dnf makecache` when they change the source.

It's still unclear to me if Fedora will run "makecache" automatically for users.

Reply:
> It seems the babies need to run `sudo dnf makecache` when they change the source.

Yes, it's mentioned at the end of the USTC help page, but I'm afraid it may be forgotten.

Reply:
Done in fd785862a1cf3b9ddc787327fea94a82277c64c4.

Note that I don't put it at the end of the subsection, because users may run the above commands first and then follow the USTC guide. It's better to follow the USTC guide first, and then try some `dnf` commands.